src/compass/utils/runconfig.py

```diff
@@ -199,21 +263,18 @@ def get_ref_radar_grid_info(ref_path, burst_ids):
     if not rdr_grid_files:
         raise FileNotFoundError(f'No reference radar grids not found in {ref_path}')
 
-    ref_rdr_grids ={}
-    for burst_id in burst_ids:
-        b_id_rdr_grid_files = [f for f in rdr_grid_files if burst_id in f]
+    b_id_rdr_grid_files = [f for f in rdr_grid_files if burst_id in f]
 
-        if not b_id_rdr_grid_files:
-            raise FileNotFoundError(f'Reference radar grid not found for {burst_id}')
+    if not b_id_rdr_grid_files:
+        raise FileNotFoundError(f'Reference radar grid not found for {burst_id}')
 
-        if len(b_id_rdr_grid_files) > 1:
-            raise FileExistsError(f'More than one reference radar grid found for {burst_id}')
+    if len(b_id_rdr_grid_files) > 1:
+        raise FileExistsError(f'More than one reference radar grid found for {burst_id}')
 
-        ref_rdr_path = os.path.dirname(b_id_rdr_grid_files[0])
-        ref_rdr_grid = file_to_rdr_grid(b_id_rdr_grid_files[0])
-        ref_rdr_grids[burst_id] = ReferenceRadarInfo(ref_rdr_path, ref_rdr_grid)
```

Comment:
Testing on the RSLC workflow for a secondary date gives the following error. It seems to be related to the changes here.

```bash
(opera) kamb:~/data-marmot/OPERA/compass_rslc>$ python -m compass.s1_cslc cslc_s1_20200517.yaml 
journal: Starting geo2rdr
Traceback (most recent call last):
  File "/home/zyunjun/tools/miniconda3/envs/opera/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/zyunjun/tools/miniconda3/envs/opera/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/zyunjun/tools/COMPASS/src/compass/s1_cslc.py", line 25, in <module>
    run(runconfig)
  File "/home/zyunjun/tools/COMPASS/src/compass/s1_cslc.py", line 12, in run
    s1_geo2rdr.run(cfg)
  File "/home/zyunjun/tools/COMPASS/src/compass/s1_geo2rdr.py", line 70, in run
    burst_path = cfg.reference_radar_info[burst_id].path
TypeError: __getitem__(): incompatible function arguments. The following argument types are supported:
    1. (self: pybind_isce3.product.RadarGridParameters, arg0: tuple) -> pybind_isce3.product.RadarGridParameters

Invoked with: <pybind_isce3.product.RadarGridParameters object at 0x7f350740a970>, 't71_iw1_b848'
```

Reply:
I will leave this to @LiangJYu who has followed the radar workflow more closely.

Reply:
Fixed with 649cbbe.

Reply:
Thank you @LiangJYu for the quick fix.